src/doc/en/developer/coding_basics.rst

```diff
@@ -255,8 +255,8 @@ The top of each Sage code file should follow this format::
     AUTHORS:
 
     - YOUR NAME (2005-01-03): initial version
```

Comment:
```suggestion
    - Your Name (2024-01-13): initial version
```

Reply:
Also change the year of the copyright from 2013 to 2024.

Reply:
Done.